### DISCUSSION TITLE
YOURLS always return 404 if I directly access short url using https

### DISCUSSION THREAD

**Discussion by fweng322** [YOURLS/YOURLS#2918] [Q&A] [answered]
Hi,

I created short URLs using YOURLS and they worked well using http://my_yourls.host/shortURL.
After upgrading to some version (forgot which) the default short URL created would become https://my_yourls.host/shortURL, however when I access directly using that URL I always got 404 error.  Changing it to http://my_yourls.host/shortURL would work.

The only worked https URL is https://my_yourls.host/admin/ which is defined in my apache conf.

I've read manuals/docs but couldn't find the reason.  It does not look like a rewrite issue.  Wondering if I need to make changes in the database.  

Could anyone please give me some hint about this?  Thanks a lot!

Franklin

**Comment by dgw** [comment, marked as answer]
Apache is probably not using the same configuration for HTTP and HTTPS, or they would behave the same.
Verify your configuration against [the wiki](https://github.com/YOURLS/YOURLS/wiki/.htaccess).
